src/uiTest/resources/project.rhino.js

```diff
@@ -421,7 +421,7 @@ function Project(underlyingProject) {
   };
 
   this.syncSelectedToParentByMerge = function (branchName) {
-    PropertiesComponent.getInstance().setValue(SHOW_MERGE_WARNING, false);
+    PropertiesComponent.getInstance().setValue(SHOW_MERGE_WARNING, false, /* default value */ true);
```

Comment:
Wow weird that the API requires the default value for **setting** (and not just getting)

Reply:
Out of curiosity, do you have any idea why it'd be designed like that? 🤔

Reply:
No idea, saving memory? Just a very error-prone design.